tests/Agent/UnitTests/Core.UnitTest/Wrapper/AgentWrapperApi/DistributedTracing/DistributedTracePayloadHandlerTests.cs

```diff
@@ -762,6 +764,53 @@ public void W3C_InsertDistributedTraceHeaders_ExcludeNewRelicHeader()
             Mock.Assert(() => _agentHealthReporter.ReportSupportabilityTraceContextCreateSuccess(), Occurs.Once());
         }
 
+        //This test makes sure string presentation for Priority in a tracestate is culture independent.
+        [TestCase(1.123000f, "1.123")]
+        public void W3C_InsertDistributedTraceHeaders_CultureIndependent_PriorityInRightFormat(float testPriority, string expectedPriorityString)
```

Comment:
By making sure string presentation for Priority in a tracestate is culture independent, we will never get non ascii character for the string presentation.